cmd/mysqldef/mysqldef_test.go

```diff
@@ -995,6 +995,40 @@ func TestMysqldefEnumValues(t *testing.T) {
 	assertApplyOutput(t, createTable, nothingModified)
 }
 
+func TestMysqldefSpatialValuesWithSRIDAndIndex(t *testing.T) {
```

Comment:
I'm trying to move SQL-level tests, at least newly added ones, to `cmd/mysqldef/tests.yml`, leaving only CLI-level tests in the Go code. I think this can be tested there too.